cli/fathom_web/commands/list.py

```diff
@@ -32,12 +32,18 @@ def main(in_directory, base_dir, recursive, out_file):
     if out_file is not None:
         filenames_to_save = []
 
+    there_were_no_files = True
     for file in path_iterator:
+        there_were_no_files = False
         relative_path = file.relative_to(base_dir)
         print(relative_path)
 
         if out_file is not None:
             filenames_to_save.append(relative_path.as_posix() + '\n')
 
+
     if out_file is not None:
-        out_file.writelines(filenames_to_save)
+        if there_were_no_files:
+            print(f'No .html files found in {in_directory}. Did not create {out_file}.')
```

Comment:
After trying this out, it looks like `out_file` here is not a string, but a [`click.File`](https://click.palletsprojects.com/en/7.x/api/#click.File), so when I trigger this if block, I get: `No .html files found in dummy. Did not create <unopened file 'training.txt' w>.` Is it possible just to get the filename string? Otherwise we could just say "Did not create the output file."